src/app/app.component.ts

```diff
@@ -0,0 +1,75 @@
+import { Component, Input } from '@angular/core';
+import { Card } from './Card';
+import { CARDS } from './mock-cards';
+
+@Component({
+  selector: 'app-root',
+  templateUrl: './app.component.html',
+  styleUrls: ['./app.component.css'],
+})
+export class AppComponent {
+  secondCards: Card[] = CARDS;
+  id: string = this.createUUID();
+  cards: Card[] = CARDS;
+  innerNumber: number = this.randomNumber();
+  card: any = {};
+
+  title = 'Task-Cards';
+
+  deleteCard(id: string) {
+    this.cards = this.cards.filter((item) => item.id !== id);
+
+    return this.cards;
+  }
+
+  createCard() {
+    this.card = {
+      id: this.createUUID(),
+      innerNumber: this.randomNumber(),
+    };
+    return this.card;
+  }
+
+  sortCard() {
+    this.sortObjectArray(this.cards);
+  }
+
+  addCard() {
+    this.cards.push(this.createCard());
+  }
+
+  draw() {
+    for (let i = 0; i < this.cards.length; i++) {
+      this.cards.push(this.secondCards[i]);
+    }
+  }
+
+  createUUID() {
+    var dt = new Date().getTime();
+    var uuid = 'xxxxxxxx-xxxx-4xxx-yxxx-xxxxxxxxxxxx'.replace(
+      /[xy]/g,
+      function (c) {
+        var r = (dt + Math.random() * 16) % 16 | 0;
+        dt = Math.floor(dt / 16);
+        return (c == 'x' ? r : (r & 0x3) | 0x8).toString(16);
+      }
+    );
+    return uuid;
+  }
+
+  randomNumber() {
```

Comment:
Move this to helper file and make randomNumber function work with min and max params